FullStoryMiddlewareExample/Base.lproj/Main.storyboard

```diff
@@ -1,24 +1,73 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would add some alignment constraints to these buttons so that they stay in an expected place. Not a blocker since this is just a sample app and would be run for a quick inspection. But this would be nice to have.